common/Environments/CustomControls/CardBody.xaml

```diff
@@ -74,6 +74,7 @@
                         Visibility="{x:Bind ComputeSystemOperationStatus, Mode=OneWay, Converter={StaticResource EmptyStringToCollapsedConverter}}"
                         Text="{x:Bind CardState, Converter={StaticResource CardStateToLocalizedTextConverter}, Mode=OneWay}"
                         Style="{StaticResource CardBodyStateTextBlockStyle}"
+                        TextWrapping="Wrap"

```

Comment:
Do you want "Wrap" or "WrapWholeWords"?

Reply:
We are using Wrap in more places. Is there a convention?

Reply:
Whatever looks nice.

Reply:
It depends on whether it makes sense to let a word be cut off in the middle. I would argue that usually, you don't want that. https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.textwrapping